src/features/HelpAndKnowledge/SolutionsHelp/solutionsMap.tsx

```diff
@@ -608,6 +588,26 @@ export const helpSolutions: HelpSolutionBaseType[] = [
       )
     }
   },
+  {
```

Comment:
Was this change (which appears to just be ordering?) necessary for any of the fixes @StevenWadeOddball / @patrickseguraoddball ?

Reply:
Ah yes -- seems it was related to the alphabetical (expected) ordering!

Reply:
Thanks Clay! Yes that is correct. I thought that the code ordered it, I didn't realize the order came from the list itself!